src/components/SelectionList/TransactionListItem.tsx

```diff
@@ -68,6 +69,23 @@ function TransactionListItem<TItem extends ListItem>({
     const merchant = getMerchant();
     const typeIcon = getTypeIcon(transactionItem.type);
 
+    const receiptCell = (isHovered = false) => (
+        <View style={[StyleUtils.getWidthAndHeightStyle(variables.h36, variables.w40), StyleUtils.getBorderRadiusStyle(variables.componentBorderRadiusSmall), styles.overflowHidden]}>
+            <ReceiptImage
+                source={transactionItem?.receipt?.source ?? ''}
```

Comment:
We need to apply `tryResolveUrlFromApiRoot` to the receipt source before passing to `source` so that URL is resolved for non production environments. Otherwise, the thumbnail images does not get shown in development/staging environments as identified [here](https://github.com/Expensify/App/pull/41957#issuecomment-2111634722). Something like this can solve this problem:
```
	const thumbnailSource = tryResolveUrlFromApiRoot(transactionItem?.receipt?.source ?? '');  
     …
	source={thumbnailSource}
```

Reply:
done